e2e/test/scripts/delete_cluster.sh

```diff
@@ -5,20 +5,9 @@ set -o pipefail
 set -o nounset
 set -x
 
-cluster_name="$1"
-terraform apply -destroy -auto-approve
-
-rm cluster.tf
-
-rm -rf test/.terraform
-
-if [[ -d ".terraform" ]]
-then
-    rm -rf .terraform
-fi
-
-if [[ -d "terraform.tfstate.d" ]]
-then
-    rm -rf terraform.tfstate.d
-fi
+export CLUSTER_NAME="$1"
+export KUBECONFIG="$(pwd)/${CLUSTER_NAME}-management.conf"
 
+timeout 5m kubectl delete linodemachine --all --wait
+timeout 5m kubectl delete linodecluster --all --wait
+timeout 5m kind delete cluster -n ${CLUSTER_NAME}
```

Comment:
nitpick: Should the local cluster deletion be handled through ctlptl rather than directly with `kind`?